src/pagination/renderer/ScrollBulletRenderer.ts

```diff
@@ -95,6 +90,29 @@ class ScrollBulletRenderer extends Renderer {
     const prevClassRegex = new RegExp(`^${prevClassPrefix}`);
     const nextClassRegex = new RegExp(`^${nextClassPrefix}`);
 
+    if (renderActiveBullet) {
+      const prevBullet = bullets[prevIndex];
+      if (prevBullet) {
+        const newBullet = this._createBulletFromString(
```

Comment:
There seems to be a task of creating an element every time it changes. How about caching?

Reply:
Well, that would be good if the rendered result is consistent.
As `renderActiveBullet` takes two parameters, `className` and `index`, I assumed that every render result can differ, and also the options can change at any time so I made it like that. I agree that it's not the best way of doing it.